distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column.enso

```diff
@@ -912,6 +918,62 @@ type Column
         _ = [type, format, on_problems]
         Error.throw <| Unsupported_Database_Operation.Error "`Column.parse` is not implemented yet for the Database backends."
 
+    ## UNSTABLE
```

Comment:
Lets mark as PRIVATE for now then so doesn't show up until true implementation done.

Reply:
Ok

Reply:
Although this implementation basically works, so I think it could be used. It's not tested perfectly, but basic stuff works. Just not in-memory.